de1plus/dui.tcl

```diff
@@ -6091,14 +6735,25 @@ namespace eval ::dui {
 		#	-tags: a label that allows to access the created canvas items
 		#	-style: to apply the default aspects of the provided style
 		#	-aspect_type: to query default aspects for type different than "text"
-		#	-compatibility_mode: set to 0 to be backwards-compatible with add_de1_text calls (don't apply aspects,
+		#	-compatibility_mode: set to 1 to be backwards-compatible with add_de1_text calls (don't apply aspects,

```

Comment:
should we really break the "API" here? if there are zero users of this in the app and usual plugins i am totally fine with it, else not loving it